prql-compiler/src/parser.rs

```diff
@@ -1866,6 +1880,14 @@ join `my-proj`.`dataset`.`table`
         )
         .is_err());
 
+        assert!(parse(
+            r###"
+        prql dialect:bigquery version:"2"
```

Comment:
```suggestion
        prql dialect:bigquery version:"25"
```

So we don't get a surprising failure when we release a new version, for a long time :)